src/services.py

```
@@ -102,3 +102,40 @@ def get_service_risk_resource(service_id: str):
+    return api_caller.call_get('/services/types')
+
+@service_mcp.tool(annotations={"readOnlyHint": True, "title": "Get Service Types"})
+def get_service_types():
+    """
+    Gets the list of service types
+    :return: list of service types
+    """
+    return api_caller.call_get('/services/types')
+
+@service_mcp.tool(annotations={"readOnlyHint": False, "title": "Create Service"})
+def create_service(name: str, description: str, type: str = "service", url: str = None, tier: int = 3):
```

Ruff (0.15.7)

[error] Function argument `type` is shadowing a Python builtin

(A002)